src/components/NewMessageModal.vue

```diff
@@ -181,6 +185,19 @@ export default {
 				showError(t('mail', 'Could not discard message'))
 			}
 		},
+		convertEditorBody(composerData) {
+			if (composerData.editorBody !== null) {
+				return composerData.editorBody
+			}
+			if (composerData.body === null) {
+				return ''
+			}
+			if (composerData.body.format !== 'html') {
+				return toHtml(composerData.body).value
+			} else {
+				 return composerData.body.value
+			}
```

Comment:
```suggestion
			toHtml(composerData.body).value
```

This is the change I suggested.

Leave it as is if it doesn't make sense. It's just a nitpick.